### PR TITLE
Update commit message guidance to mention skipping ci

### PR DESCRIPTION
**Pull request by DanHeidinga**
[skip ci]

Signed-off-by: Dan Heidinga <daniel_heidinga@ca.ibm.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -69,6 +69,10 @@ copying resources into the body of the commit over providing external links.
 Structure large commit messages with headers, references etc. Remember, however,
 that the commit message is always going to be rendered in plain text.
 
+Please add `[skip ci]` to the commit message when the change doesn't require a 
+compilation, such as documentation only changes.  This is required to be good
```
Comment (mstoodle) [inline CONTRIBUTING.md]: Maybe ...such as documentation only changes, to avoid unnecessarily wasting the project's build resources.

?

**Comment by AdamBrousseau** [comment]
Default skip phrase for Jenkins is `[skip ci]` I think travis will accept either `[skip ci]` or `[ci skip]` based on https://github.com/travis-ci/travis-ci/issues/911

**Comment by DanHeidinga** [comment]
Updated to use your wording @mstoodle

**Comment by genie-openj9** [comment]
Can one of the admins verify this patch?

**Comment by DanHeidinga** [comment]
FYI - opened a similar PR for OMR: https://github.com/eclipse/omr/pull/1702

**Comment by AdamBrousseau** [comment]
add to whitelist

**Pull request merged**